src/jsonValidator.js

```diff
@@ -0,0 +1,17 @@
+import { createError } from './error.js';
+
+export const jsonIsValid = (jsonString, errors) => {
+  try {
+    const checkJson = JSON.parse(jsonString);
+  } catch (error) {
+    errors.push(createError("invalid-json"));
+    return {
+      valid : false,
+      errors
```

Comment:
Can someone provide an example of usage of this function?
I'm having a hard time trying to understand how to use `jsonIsValid()` and the return is an object.

Reply:
I agree. Is the purpose of this function to be a validator or a boolean check?

Reply:
the `errors` variable is being mutated in the code. We wouldn't need to return it since it's already being changed. The return from this function should be Boolean. 
Thoughts?